packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.tsx

```diff
@@ -1,32 +1,25 @@
-import {QuerySetActionCreators, Suggestion} from '@coveo/headless';
-import {Component, h, Prop, State} from '@stencil/core';
+import {loadInsightSearchActions} from '@coveo/headless/insight';
+import {Component, Element, h, Prop, State} from '@stencil/core';
 import {
   buildInsightSearchBox,
   InsightSearchBox,
   InsightSearchBoxState,
-  loadInsightQuerySetActions,
+  InsightSuggestion,
 } from '..';
```

Comment:
Barrel imports; do we really want them?

Reply:
They are there because of name collision in different packages in Headless, when imported into a single Atomic bundle.

d.ts file would generate some weird names otherwise.

As for the previous import that mot modified: yes, we certainly want to import from `/headless/insight`, not `/headless`: that's pollution and incorrect.